### PR TITLE
Update allowed admin groups

### PR DESCRIPTION
**Pull request by aktech**
## Reference Issues or PRs

Fixes https://github.com/nebari-dev/nebari/issues/2436

Due to a recent change, adding `jupyterhub_admin` role to a user doesn't makes it admin for JupyterHub, this PR fixes that.

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

### CODE REVIEW

**Comment by viniciusdc** [comment]
This needs to be tested; based on the changes, I don't foresee any problems. but its good to double check

**Comment by aktech** [comment]
> This needs to be tested; based on the changes, I don't foresee any problems. but its good to double check

We tested this on Quansight deployment and a client deployment. This is a fix for the consequence of: https://github.com/nebari-dev/nebari/commit/6a83adad44ac096f8723ec753035fd3a8ed39296

This could be soon removed, when we have roles syncing and proper permissions model, which is WIP. This PR is just be bring back the old behaviour meanwhile.

File: src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/main.tf [hunk 1]
```diff
@@ -159,8 +159,8 @@ resource "helm_release" "jupyterhub" {
             login_service        = "Keycloak"
             username_claim       = "preferred_username"
             claim_groups_key     = "groups"
-            allowed_groups       = ["/analyst", "/developer", "/admin"]
-            admin_groups         = ["/admin"]
+            allowed_groups       = ["/analyst", "/developer", "/admin", "jupyterhub_admin", "jupyterhub_developer"]
+            admin_groups         = ["/admin", "jupyterhub_admin"]
```
Comment (viniciusdc) [inline src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/main.tf:163]: Hi @aktech wouldn't this create conda-store namespaces and EFS share files for those then when a users receives such group?

**Comment by viniciusdc** [comment]
@krassowski, if you have some time today, could you just double-check that those permissions are enough?

**Comment by viniciusdc** [comment]
As this is a release blocker I will go ahead an merge this PR, and any extra thoughts can be done in a separate PR after the first RC

**Review by viniciusdc — APPROVED** [review]
LGTM

**Pull request merged**